spring-kafka/src/main/java/org/springframework/kafka/config/TopicBuilder.java

```diff
@@ -20,12 +20,14 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import java.util.Optional;
 
 import org.apache.kafka.clients.admin.NewTopic;
 import org.apache.kafka.common.config.TopicConfig;
 
 /**
- * Builder for a {@link NewTopic}.
+ * Builder for a {@link NewTopic}. Since 2.3 partitions and replicas default to
```

Comment:
Probably `2.6`? You don’t mention back-porting. Plus looks like this class is since `2.3` per se...